joss/paper.md

```diff
@@ -51,20 +51,20 @@ _tfaip_ resolves the following different aspects in an elegant and robust way.
+This prevents users from making mistakes by setting parameters without effect for the current configuration (e.g., setting the factor for an Adam optimizer even though RMSprop was selected).
 The default CLI of _tfaip_ provides commands to adapt various hyper-parameters such as the learning rate and its schedule, the optimizer, logging, debugging, profiling, or early stopping.
 Furthermore, each component of the scenario can itself be fully customized which allows, for example, to dynamically configure the network architecture, e.g., by inserting layers or changing their parameters.
 This feature helps researchers to set up various experiments for example to optimize hyper-parameters or test novel ideas.
 
-In comparison to other frameworks such as Tensorflow, _tfaip_ requires users to implement their scenarios in object-oriented programming and encourages them to annotate their code with types.
-This is particularly sensible if the scenarios and thus their code basis becomes larger since it leads to a clean, structured, modularized, and readable code preventing bad code style and facilitates maintenance.
+In comparison to other frameworks such as Tensorflow, _tfaip_ requires users to implement their scenarios in an object-oriented way and encourages them to annotate their code with types.
+This is particularly advantageous for larger, more complex scenarios where the codebase grows accordingly. Using _tfaip_ leads to a clean, structured, modularized, and readable code limiting poor coding styles and facilitating maintenance.
 In practice, each scenario is created by implementing predefined interfaces (e.g., loss-function or the graph construction).
 
 During research and development, a tedious step is data preparation which often comprises the conversion of data into the format required by the framework.
-The Tensorflow-backed of _tfaip_ allows integrating Python code in the data pipeline which is however not run (truly) in parallel by multiple processes and results quite often in a bottleneck.
-To speed-up Tensorflow, a user has to transform Python into Tensorflow operations which is laborious, partly even impossible, and complicates debugging.
-_tfaip_ tackles this issue by providing a sophisticated pipeline setup based on so-called data processors which apply simple transformation operations in pure Python code and are automatically executed in parallel.
+The Tensorflow-backed _tfaip_ allows integrating Python code in the data pipeline which is however not run (truly) in parallel by multiple processes and results quite often in a bottleneck.
+To speed-up Tensorflow, a user has to transform Python into Tensorflow operations which is laborious, and sometimes even impossible, and complicates debugging.
+_tfaip_ tackles this issue by providing a sophisticated pipeline setup based of 'data processors' which apply simple transformation operations in pure Python code and are automatically executed in parallel.
```

Comment:
I'd suggest changing this to double quotes and "based on":

> pipeline setup based on "data processors"

Reply:
```suggestion
_tfaip_ tackles this issue by providing a sophisticated pipeline setup based on "data processors" which apply simple transformation operations in pure Python code and are automatically executed in parallel.
```